src/components/autocompleteInput.js

```diff
@@ -330,62 +328,67 @@
      */
     /* eslint-disable no-underscore-dangle */
 
-    const numberDebouncedInputValue = parseFloat(debouncedInputValue, 10);
-
-    // There is an empty string as fallback, because empty label is not a valid value to search on
-    const textDebouncedInputValue =
```

Comment:
I see you have removed the textDebouncedInputValue that was just recently added by Jordy, is that correct?